composer.json

```diff
@@ -45,10 +45,5 @@
         "psr-4": {
             "Amp\\Test\\Artax\\": "test"
         }
-    },
-    "config": {
-        "platform": {
-            "php": "7.0.0"
```

Comment:
This option will force composer to use dependencies compatible for 7.0 which is not good when a different version of PHP is used on a target machine.